packages/backend/src/graphql/mutations/update-flow-transfer-status.ts

```diff
@@ -0,0 +1,25 @@
+import { IFlowTransferStatus } from '@plumber/types'
+
+import FlowTransfer from '@/models/flow-transfers'
+
+type Params = {
+  input: {
+    id: string
+    status: IFlowTransferStatus
```

Comment:
nit: i'm 80% sure this is a lie; Apollo will not validate this. If we wanted to be really careful, we should type this as `string` and use a type guard

Reply:
i think it will be a bad user input if an invalid `status` is given?